code/index.html

```diff
@@ -1,30 +1,158 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@100&family=Lato:wght@100&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+
+  <!-- dont forget to add a css file and link it here! -->
+</head>
+
+<body>
+  <!-- video or image as a header is cool :) -->
+
+
+  <div class="header">
+
+    <video autoplay loop muted class="header__video">
+      <source src="Sky - 3186.mp4">
+    </video>
+    <div class="overlay">
+
+      <button class="knapp"></button>
+
+      <h1>HEAVEN</h1>
+      
+
+    </div>
+
+  </div>
+
+
+
+
+
+  <section class="content">
+    <section class="content-item1">
+      <h1>Sign In</h1>
+
+
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+
+        <div class="form-group">
+          <label for="firstName">First name</label>
+          <input class="w3-input" id="firstName" type="text" placeholder="Type your first name" name="firstNameValue"
+            required />
+        </div>
+
+        <div class="form-group">
+          <label for="lastName">Last name</label>
+          <input class="w3-input" id="lastName" type="text" placeholder="Type your last name" name="lastNameValue"
+            required />
+        </div>
+
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input class="w3-input" id="email" type="email" placeholder="Type your email" name="emailValue" required />
+        </div>
+        <section class="radio">
+
+          <label class="container">Female
+            <input type="radio" name="radio">
+
+          </label>
+          <label class="container">Male
+            <input type="radio" name="radio">
+
+          </label>
+          <label class="container">Other
+            <input type="radio" name="radio">
+
+          </label>
+
+
+        </section>
+
+        <button class="form-submit-one" type="submit"></button>
```

Comment:
You need to inform the user of what this button will do when it's clicked. It should say 'send' or 'submit', or something like that.

Reply:
I put in a > in the button